staging/src/k8s.io/bootstrap/OWNERS

```diff
@@ -0,0 +1,15 @@
+approvers:
+- caesarxuchao
+- deads2k
+- lavalamp
+- liggitt
+- smarterclayton
+- sttts
+reviewers:
+- caesarxuchao
```

Comment:
Please add sig-cluster-lifecycle folks to the approvers+reviewers list.

Reply:
added sig-cluster-lifecycle folks to the approvers+reviewers list